app/models/name/resolve.rb

```diff
@@ -179,7 +179,9 @@ def self.fix_capitalized_species_epithet(str)
     str2 = str.sub(/ [A-Z]/, &:downcase)
 
     # Return corrected name if that name exists, else keep original name.
-    if Name.where("search_name = ? OR text_name = ?", str2, str2).present?
+    # if Name.where("search_name = ? OR text_name = ?", str2, str2).present?
+    if Name.where(Name[:search_name].eq(str2).
+                  or(Name[:text_name].eq(str2))).present?
```

Comment:
Isn't there an "or" in AR?

`Name.where(search_name: str2).or(text_name: str2)`